dts/msm8974/msm8974-samsung-hlte-r07.dts

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+#include <lk2nd.h>
+
+/ {
+	// This is used by the bootloader to find the correct DTB
+	qcom,msm-id = <0x7E01FF01 7 0>, <0xB801FF01 7 0>, <0xB901FF01 7 0>;
```

Comment:
```suggestion
	qcom,msm-id = <0x7E01FF01 7 0>;
```

the three in here were: msm8974(126), apq8074(184), msm8274(185), per the highest byte and qcom,ids.h. I don't think you need apq and the 3g-only chip here...

Reply:
Got it.
Hadn't looked through what they were back then; just kept all three as they were in the dts.